### PR TITLE
[13.0][MIG] base_maintenance

### PR DESCRIPTION
**Pull request by NuriaXifre**
Migration of the module base_maintenance to v13.
cc ~ @ForgeFlow

### CODE REVIEW

**Review by etobella — APPROVED** [review]
LGTM :+1: No real changes

File: base_maintenance/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Base Maintenance",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "author": "ForgeFlow, " "Odoo Community Association (OCA)",
```
Comment (AdriaGForgeFlow) [inline base_maintenance/__manifest__.py]: Remove extra double quotes, please.

**Review by AdriaGForgeFlow — APPROVED** [review]
LGTM :+1: 

Just small comment below

File: base_maintenance/models/maintenance_team.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class MaintenanceTeam(models.Model):
+    _inherit = "maintenance.team"
+
+    active = fields.Boolean(default=True)
```
Comment (AdriaGForgeFlow) [inline base_maintenance/models/maintenance_team.py]: This field is introduced in Odoo 12.0, no need to be declared here anymore

**Comment by etobella** [comment]
@NuriaMForgeFlow There is issues with the pre-commit. Can you check it please :wink:

File: base_maintenance/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Base Maintenance",
+    "version": "13.0.1.0.0",
+    "author": "ForgeFlow," "Odoo Community Association (OCA)",
```
Comment (LoisRForgeFlow) [inline base_maintenance/__manifest__.py]: Better if this is only one string. `"ForgeFlow, Odoo Community Association (OCA)",`

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Pull request merged**